ArduCopter/Log.cpp

```diff
@@ -75,9 +75,7 @@ void Copter::Log_Write_Control_Tuning()
 // Write an attitude packet
 void Copter::Log_Write_Attitude()
 {
-    Vector3f targets = attitude_control->get_att_target_euler_cd();
-    targets.z = wrap_360_cd(targets.z);
-    ahrs.Write_Attitude(targets);
+    ahrs.Write_Attitude(attitude_control->get_att_target_euler_rad());
```

Comment:
You appear to be passing the targets in in radians in several places here now.

Pretty sure you need to convert to degrees here somewhere...

You can multiply the vector by `RAD_TO_DEG`...

Reply:
Thanks, fixed

Reply:
OK, so you've fixed this in `Write_Attitude`.

Two issues there.

tridge probably won't like radians in the interface.

You haven't changed the parameter names to add `_rad` on the end

Reply:
Fixed